src/components/KCollapse/KCollapseItem.vue

```diff
@@ -0,0 +1,176 @@
+<template>
```

Comment:
issue: Again, this new child component is adding a lot of complexity; I think this should be handled by providing generic slots whereby the host app can inject whatever content is needed in the visible/hidden slots instead of such a rigid layout.